Simplenote/SPAppDelegate.m

```diff
@@ -325,6 +326,7 @@ - (void)onboardingDidFinish:(NSNotification *)notification
 
 - (void)applicationWillResignActive:(UIApplication *)application
 {
+    [self.tagListViewController removeKeyboardObservers];
```

Comment:
I believe I attempted using that, but the notification wouldn't fire until after the code was run in the app delegate. We need the listener to go away before the app adds the new UIWindow for the Pin lock.